src/Surfnet/StepupRa/RaBundle/Security/Authentication/Token/SamlToken.php

```diff
@@ -61,12 +79,12 @@ public function getLoa()
 
     public function serialize()
     {
-        return serialize([parent::serialize(), $this->loa,]);
+        return serialize([parent::serialize(), $this->loa, $this->institutionConfigurationOptions]);
```

Comment:
also, might be worth considering using an RA specific variant here, so that when changing the class in a BC breaking way, we don't invalidate all sessions after deploy (albeit that might even be best, hrms) - in any case, think about it :stuck_out_tongue_winking_eye: